XRPLib/imu.py

```diff
@@ -110,38 +112,112 @@ def _r_w_reg(self, reg, dat, mask):
         self.rb[0] = (self.rb[0] & mask) | dat
         self._setreg(reg, self.rb[0])
 
-    def _mg(self, reg):
-        return round(self._int16(self._get2reg(reg)) * 0.061 * self._scale_a_c)
+    def is_connected(self):
```

Comment:
Very minor thing, just for organization purposes, is_connected and reset should be moved down to below the comment labeling that section as public facing methods

Reply:
Woah! I'm blind, didn't realize the functions were divided up 😅 I'll move those into the public section.

While we're on this topic, any reason `_get_gyro_x_rate()` (y and z too), `_get_gyro_rates()`, and `_get_acc_gyro_rates()` can't be public? `get_acc_x()` (y and z too) and `get_acc_rates()` are public. I understand that users will usually get gyro data from `get_yaw()` (and others), but it feels weird to me for the accelerometer functions to be public, and the gyro functions to be private, especially since they're all very similar to each other. Eh, probably doesn't matter, but figured I'd suggest it.

Also just noticed `get_acc_rates()` probably shouldn't have `rate` in the name, since it's not a rate like the gyro measurements are.

Reply:
I agree with you here, I think they should be public. Ansel had made them private somewhere down the line but I think it's a better learning tool if we expose the fact that the gyro returns a rate instead of just the raw value; I just haven't gone through and made them public yet.

Reply:
Done!